### PR TITLE
feat(Buttons): ColorModes + Props + Refactor + Refs Oh My! [GM-211] [REACH-856] [GM-192]

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

This PR defaults the `mode` prop of all buttons to the currently active ColorMode instead of `light`.  The prop can still be configured and overridden if need however.

* Buttons now use `ButtonBase` to match with `Anchor`.
* Removes extra tagged templates to reduce Emotion Label spam and reduces internal style overhead.
* Button Colors are now serialized as variables inside of the base component depending on which mode they are.
* Buttons will now default to the current ColorMode if they have not been passed a prop.
* Also fixes a prop forwarding bug with CTAButton where inner component was not getting props forwarded.
* Adds all button types to the ColorMode example (small and large).
* Buttons now will use the current mode in context unless specified otherwise. Since all dark mode buttons are specifically configured this way there should be no changes to ANY existing button colors.
* Buttons now come with a few system props to make life easier including spacing layout and positioning. See prop reference here https://gamut.codecademy.com/storybook/?path=/docs/foundations-system-props--layout. Please note that these will cause issues with behavior if used incorrectly. Changes to display and padding may break button styles in some cases.
* All Buttons now accept a ref!

Example: 

https://user-images.githubusercontent.com/52927224/115595497-fe512880-a2a4-11eb-8daf-81061e69668f.mp4

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/N6TfoPI5OA7GanX6Zl8nWj/Buttons-%26-Links
- [x] Related to JIRA ticket: GM-211 GM-192 REACH-856
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.3-alpha.acbb93.0<br/>@codecademy/gamut-kit@0.3.39-alpha.acbb93.0<br/>@codecademy/gamut-labs@14.0.1-alpha.acbb93.0<br/>@codecademy/gamut-styles@9.5.4-alpha.acbb93.0<br/>@codecademy/gamut-tests@2.3.15-alpha.acbb93.0<br/>@codecademy/gamut@32.0.1-alpha.acbb93.0<br/>@codecademy/konamimojisplosion@0.2.15-alpha.acbb93.0<br/>@codecademy/markdown-overrides@0.5.27-alpha.acbb93.0<br/>@codecademy/styleguide@36.0.1-alpha.acbb93.0<br/>@codecademy/variance@0.9.1-alpha.acbb93.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6080523108f4b00e7544ea0a--gamut-preview.netlify.app>https://6080523108f4b00e7544ea0a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6080523108f4b00e7544ea0a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.3-alpha.07d22c.0<br/>@codecademy/gamut-kit@0.3.39-alpha.07d22c.0<br/>@codecademy/gamut-labs@14.0.1-alpha.07d22c.0<br/>@codecademy/gamut-styles@9.5.4-alpha.07d22c.0<br/>@codecademy/gamut-tests@2.3.15-alpha.07d22c.0<br/>@codecademy/gamut@32.0.1-alpha.07d22c.0<br/>@codecademy/konamimojisplosion@0.2.15-alpha.07d22c.0<br/>@codecademy/markdown-overrides@0.5.27-alpha.07d22c.0<br/>@codecademy/styleguide@36.0.1-alpha.07d22c.0<br/>@codecademy/variance@0.9.1-alpha.07d22c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608055237c60351bfeb25ebc--gamut-preview.netlify.app>https://608055237c60351bfeb25ebc--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608055237c60351bfeb25ebc>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.3-alpha.744f7b.0<br/>@codecademy/gamut-kit@0.3.39-alpha.744f7b.0<br/>@codecademy/gamut-labs@14.0.1-alpha.744f7b.0<br/>@codecademy/gamut-styles@9.5.4-alpha.744f7b.0<br/>@codecademy/gamut-tests@2.3.15-alpha.744f7b.0<br/>@codecademy/gamut@32.0.1-alpha.744f7b.0<br/>@codecademy/konamimojisplosion@0.2.15-alpha.744f7b.0<br/>@codecademy/markdown-overrides@0.5.27-alpha.744f7b.0<br/>@codecademy/styleguide@36.0.1-alpha.744f7b.0<br/>@codecademy/variance@0.9.1-alpha.744f7b.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.3-alpha.ca2998.0<br/>@codecademy/gamut-kit@0.3.39-alpha.ca2998.0<br/>@codecademy/gamut-labs@14.0.1-alpha.ca2998.0<br/>@codecademy/gamut-styles@9.5.4-alpha.ca2998.0<br/>@codecademy/gamut-tests@2.3.15-alpha.ca2998.0<br/>@codecademy/gamut@32.0.1-alpha.ca2998.0<br/>@codecademy/konamimojisplosion@0.2.15-alpha.ca2998.0<br/>@codecademy/markdown-overrides@0.5.27-alpha.ca2998.0<br/>@codecademy/styleguide@36.0.1-alpha.ca2998.0<br/>@codecademy/variance@0.9.1-alpha.ca2998.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6080604108f4b02b9f44e47d--gamut-preview.netlify.app>https://6080604108f4b02b9f44e47d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6080604108f4b02b9f44e47d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.49-alpha.c9b89f.0<br/>@codecademy/gamut-labs@14.4.1-alpha.c9b89f.0<br/>@codecademy/gamut@32.3.1-alpha.c9b89f.0<br/>@codecademy/styleguide@36.4.1-alpha.c9b89f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6086e899046e6c27723cd7f5--gamut-preview.netlify.app>https://6086e899046e6c27723cd7f5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6086e899046e6c27723cd7f5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.50-alpha.27fde6.0<br/>@codecademy/gamut-labs@14.4.2-alpha.27fde6.0<br/>@codecademy/gamut@32.3.2-alpha.27fde6.0<br/>@codecademy/styleguide@36.4.2-alpha.27fde6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608730a1ac796615b12de511--gamut-preview.netlify.app>https://608730a1ac796615b12de511--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608730a1ac796615b12de511>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.51-alpha.eeda5c.0<br/>@codecademy/gamut-labs@14.4.3-alpha.eeda5c.0<br/>@codecademy/gamut@32.3.3-alpha.eeda5c.0<br/>@codecademy/styleguide@36.4.3-alpha.eeda5c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60874712afcd251272d5d24f--gamut-preview.netlify.app>https://60874712afcd251272d5d24f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60874712afcd251272d5d24f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.51-alpha.225332.0<br/>@codecademy/gamut-labs@14.4.3-alpha.225332.0<br/>@codecademy/gamut@32.3.3-alpha.225332.0<br/>@codecademy/styleguide@36.4.3-alpha.225332.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608749d7b0b2f5178ac13d99--gamut-preview.netlify.app>https://608749d7b0b2f5178ac13d99--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608749d7b0b2f5178ac13d99>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.51-alpha.036db8.0<br/>@codecademy/gamut-labs@14.4.3-alpha.036db8.0<br/>@codecademy/gamut@32.3.3-alpha.036db8.0<br/>@codecademy/styleguide@36.4.3-alpha.036db8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60875d2804bd593d19fbf8dc--gamut-preview.netlify.app>https://60875d2804bd593d19fbf8dc--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60875d2804bd593d19fbf8dc>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.51-alpha.0343fb.0<br/>@codecademy/gamut-labs@14.4.3-alpha.0343fb.0<br/>@codecademy/gamut@32.3.3-alpha.0343fb.0<br/>@codecademy/styleguide@36.4.3-alpha.0343fb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6087666345b79a4321a457ae--gamut-preview.netlify.app>https://6087666345b79a4321a457ae--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6087666345b79a4321a457ae>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.4dd16b.0<br/>@codecademy/gamut-kit@0.3.52-alpha.4dd16b.0<br/>@codecademy/gamut-labs@14.4.4-alpha.4dd16b.0<br/>@codecademy/gamut-styles@9.6.2-alpha.4dd16b.0<br/>@codecademy/gamut-tests@2.3.20-alpha.4dd16b.0<br/>@codecademy/gamut@32.3.4-alpha.4dd16b.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.4dd16b.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.4dd16b.0<br/>@codecademy/styleguide@36.4.4-alpha.4dd16b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6087769a050a51526152bb03--gamut-preview.netlify.app>https://6087769a050a51526152bb03--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6087769a050a51526152bb03>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60877aef45b79a56fba457a7--gamut-preview.netlify.app>https://60877aef45b79a56fba457a7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60877aef45b79a56fba457a7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.6189eb.0<br/>@codecademy/gamut-kit@0.3.52-alpha.6189eb.0<br/>@codecademy/gamut-labs@14.4.4-alpha.6189eb.0<br/>@codecademy/gamut-styles@9.6.2-alpha.6189eb.0<br/>@codecademy/gamut-tests@2.3.20-alpha.6189eb.0<br/>@codecademy/gamut@32.3.4-alpha.6189eb.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.6189eb.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.6189eb.0<br/>@codecademy/styleguide@36.4.4-alpha.6189eb.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.9756b6.0<br/>@codecademy/gamut-kit@0.3.52-alpha.9756b6.0<br/>@codecademy/gamut-labs@14.4.4-alpha.9756b6.0<br/>@codecademy/gamut-styles@9.6.2-alpha.9756b6.0<br/>@codecademy/gamut-tests@2.3.20-alpha.9756b6.0<br/>@codecademy/gamut@32.3.4-alpha.9756b6.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.9756b6.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.9756b6.0<br/>@codecademy/styleguide@36.4.4-alpha.9756b6.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.fc8513.0<br/>@codecademy/gamut-kit@0.3.52-alpha.fc8513.0<br/>@codecademy/gamut-labs@14.4.4-alpha.fc8513.0<br/>@codecademy/gamut-styles@9.6.2-alpha.fc8513.0<br/>@codecademy/gamut-tests@2.3.20-alpha.fc8513.0<br/>@codecademy/gamut@32.3.4-alpha.fc8513.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.fc8513.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.fc8513.0<br/>@codecademy/styleguide@36.4.4-alpha.fc8513.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608839ad050a51669452b962--gamut-preview.netlify.app>https://608839ad050a51669452b962--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608839ad050a51669452b962>Deploy Logs</a></p>

File: packages/gamut/src/ButtonBase/ButtonBase.tsx [hunk 1]
```diff
@@ -1,52 +1,69 @@
...
 export type ButtonBaseElementProps = HTMLProps<
   HTMLAnchorElement | HTMLButtonElement
-> & { as?: never };
+> & {
+  as?: never;
+  ref?:
+    | ((instance: ButtonBaseElements | null) => void)
+    | MutableRefObject<ButtonBaseElements | null>
+    | null;
+};
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/ButtonBase/ButtonBase.tsx:15]: lol

File: packages/gamut/src/Button/ButtonInner.tsx [hunk 2]
```diff
@@ -1,35 +1,30 @@
...
+      content: '""',
+      display: 'inline-block',
+      width: '1px',
+      height: 1,
+      marginLeft: '-1px',
+      marginRight: '-1px',
+      verticalAlign: 'middle',
+    },
+    '> *': {
+      verticalAlign: 'middle',
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/Button/ButtonInner.tsx:27]: IIRC we used to use `flex` in other button contexts, maybe that would be good here to avoid the `*`?

**Review by JoshuaKGoldberg — APPROVED** [review]
🌮

File: packages/gamut/src/Button/ButtonInner.tsx (hunk 2 above)
Comment (codecaaron) [reply packages/gamut/src/Button/ButtonInner.tsx:27]: Yeah, I think the next step here is to convert these elements to `styled` single elements that take the `withComponent(ButtonBase)`.

I remember there were some weird alignments for `inline-flex` as it changes the flow in some containers, unclear why or if it still happens but I'm down to try just moving it to inline-flex altogether.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.8e341a.0<br/>@codecademy/gamut-kit@0.3.52-alpha.8e341a.0<br/>@codecademy/gamut-labs@14.4.4-alpha.8e341a.0<br/>@codecademy/gamut-styles@9.6.2-alpha.8e341a.0<br/>@codecademy/gamut-tests@2.3.20-alpha.8e341a.0<br/>@codecademy/gamut@32.3.4-alpha.8e341a.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.8e341a.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.8e341a.0<br/>@codecademy/styleguide@36.4.4-alpha.8e341a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608846b877283a756d21a4e9--gamut-preview.netlify.app>https://608846b877283a756d21a4e9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608846b877283a756d21a4e9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.61cc02.0<br/>@codecademy/gamut-kit@0.3.52-alpha.61cc02.0<br/>@codecademy/gamut-labs@14.4.4-alpha.61cc02.0<br/>@codecademy/gamut-styles@9.6.2-alpha.61cc02.0<br/>@codecademy/gamut-tests@2.3.20-alpha.61cc02.0<br/>@codecademy/gamut@32.3.4-alpha.61cc02.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.61cc02.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.61cc02.0<br/>@codecademy/styleguide@36.4.4-alpha.61cc02.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60884d032f810400906f63fa--gamut-preview.netlify.app>https://60884d032f810400906f63fa--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60884d032f810400906f63fa>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.880c4c.0<br/>@codecademy/gamut-kit@0.3.52-alpha.880c4c.0<br/>@codecademy/gamut-labs@14.4.4-alpha.880c4c.0<br/>@codecademy/gamut-styles@9.6.2-alpha.880c4c.0<br/>@codecademy/gamut-tests@2.3.20-alpha.880c4c.0<br/>@codecademy/gamut@32.3.4-alpha.880c4c.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.880c4c.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.880c4c.0<br/>@codecademy/styleguide@36.4.4-alpha.880c4c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60886e64500feb0d13cc6d42--gamut-preview.netlify.app>https://60886e64500feb0d13cc6d42--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60886e64500feb0d13cc6d42>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.845ef5.0<br/>@codecademy/gamut-kit@0.3.52-alpha.845ef5.0<br/>@codecademy/gamut-labs@14.4.4-alpha.845ef5.0<br/>@codecademy/gamut-styles@9.6.2-alpha.845ef5.0<br/>@codecademy/gamut-tests@2.3.20-alpha.845ef5.0<br/>@codecademy/gamut@32.3.4-alpha.845ef5.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.845ef5.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.845ef5.0<br/>@codecademy/styleguide@36.4.4-alpha.845ef5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608871045c9bfb00b8bf09f5--gamut-preview.netlify.app>https://608871045c9bfb00b8bf09f5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608871045c9bfb00b8bf09f5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.236653.0<br/>@codecademy/gamut-kit@0.3.52-alpha.236653.0<br/>@codecademy/gamut-labs@14.4.4-alpha.236653.0<br/>@codecademy/gamut-styles@9.6.2-alpha.236653.0<br/>@codecademy/gamut-tests@2.3.20-alpha.236653.0<br/>@codecademy/gamut@32.3.4-alpha.236653.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.236653.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.236653.0<br/>@codecademy/styleguide@36.4.4-alpha.236653.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608875c68673c600df92cc76--gamut-preview.netlify.app>https://608875c68673c600df92cc76--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608875c68673c600df92cc76>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.81b6f1.0<br/>@codecademy/gamut-kit@0.3.55-alpha.81b6f1.0<br/>@codecademy/gamut-labs@14.5.1-alpha.81b6f1.0<br/>@codecademy/gamut-styles@9.6.2-alpha.81b6f1.0<br/>@codecademy/gamut-tests@2.3.20-alpha.81b6f1.0<br/>@codecademy/gamut@32.4.1-alpha.81b6f1.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.81b6f1.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.81b6f1.0<br/>@codecademy/styleguide@36.5.3-alpha.81b6f1.0<br/>@codecademy/variance@0.10.1-alpha.81b6f1.0<br/></pre>

**Comment by codecaaron** [comment]
@rogervera  tagging for https://codecademy.atlassian.net/browse/REACH-856

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6088890cdd29472079a23832--gamut-preview.netlify.app>https://6088890cdd29472079a23832--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6088890cdd29472079a23832>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.2e9620.0<br/>@codecademy/gamut-kit@0.3.55-alpha.2e9620.0<br/>@codecademy/gamut-labs@14.5.1-alpha.2e9620.0<br/>@codecademy/gamut-styles@9.6.2-alpha.2e9620.0<br/>@codecademy/gamut-tests@2.3.20-alpha.2e9620.0<br/>@codecademy/gamut@32.4.1-alpha.2e9620.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.2e9620.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.2e9620.0<br/>@codecademy/styleguide@36.5.3-alpha.2e9620.0<br/>@codecademy/variance@0.10.1-alpha.2e9620.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60888edc3d1d5d1df77263bb--gamut-preview.netlify.app>https://60888edc3d1d5d1df77263bb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60888edc3d1d5d1df77263bb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.8-alpha.3dde9d.0<br/>@codecademy/gamut-kit@0.3.55-alpha.3dde9d.0<br/>@codecademy/gamut-labs@14.5.1-alpha.3dde9d.0<br/>@codecademy/gamut-styles@9.6.2-alpha.3dde9d.0<br/>@codecademy/gamut-tests@2.3.20-alpha.3dde9d.0<br/>@codecademy/gamut@32.4.1-alpha.3dde9d.0<br/>@codecademy/konamimojisplosion@0.2.20-alpha.3dde9d.0<br/>@codecademy/markdown-overrides@0.5.32-alpha.3dde9d.0<br/>@codecademy/styleguide@36.5.3-alpha.3dde9d.0<br/>@codecademy/variance@0.10.1-alpha.3dde9d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6088951f8b6aa02efbdd7f3f--gamut-preview.netlify.app>https://6088951f8b6aa02efbdd7f3f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6088951f8b6aa02efbdd7f3f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.10-alpha.15c45f.0<br/>@codecademy/gamut-kit@0.3.59-alpha.15c45f.0<br/>@codecademy/gamut-labs@15.1.2-alpha.15c45f.0<br/>@codecademy/gamut-styles@9.6.4-alpha.15c45f.0<br/>@codecademy/gamut-tests@2.3.22-alpha.15c45f.0<br/>@codecademy/gamut@32.4.3-alpha.15c45f.0<br/>@codecademy/konamimojisplosion@0.2.22-alpha.15c45f.0<br/>@codecademy/markdown-overrides@0.5.34-alpha.15c45f.0<br/>@codecademy/styleguide@36.6.2-alpha.15c45f.0<br/>@codecademy/variance@0.10.2-alpha.15c45f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608ac11c9b627d008c9ded3d--gamut-preview.netlify.app>https://608ac11c9b627d008c9ded3d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608ac11c9b627d008c9ded3d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.10-alpha.eeea1b.0<br/>@codecademy/gamut-kit@0.3.59-alpha.eeea1b.0<br/>@codecademy/gamut-labs@15.1.2-alpha.eeea1b.0<br/>@codecademy/gamut-styles@9.6.4-alpha.eeea1b.0<br/>@codecademy/gamut-tests@2.3.22-alpha.eeea1b.0<br/>@codecademy/gamut@32.4.3-alpha.eeea1b.0<br/>@codecademy/konamimojisplosion@0.2.22-alpha.eeea1b.0<br/>@codecademy/markdown-overrides@0.5.34-alpha.eeea1b.0<br/>@codecademy/styleguide@36.6.2-alpha.eeea1b.0<br/>@codecademy/variance@0.10.2-alpha.eeea1b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608ac2d72ac77600cb5dd423--gamut-preview.netlify.app>https://608ac2d72ac77600cb5dd423--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608ac2d72ac77600cb5dd423>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608b06345c9c8552ea6b1f1c--gamut-preview.netlify.app>https://608b06345c9c8552ea6b1f1c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608b06345c9c8552ea6b1f1c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.10.10-alpha.c46aa9.0<br/>@codecademy/gamut-kit@0.3.61-alpha.c46aa9.0<br/>@codecademy/gamut-labs@15.2.1-alpha.c46aa9.0<br/>@codecademy/gamut-styles@9.6.4-alpha.c46aa9.0<br/>@codecademy/gamut-tests@2.3.22-alpha.c46aa9.0<br/>@codecademy/gamut@32.5.1-alpha.c46aa9.0<br/>@codecademy/konamimojisplosion@0.2.22-alpha.c46aa9.0<br/>@codecademy/markdown-overrides@0.5.34-alpha.c46aa9.0<br/>@codecademy/styleguide@36.7.1-alpha.c46aa9.0<br/>@codecademy/variance@0.10.2-alpha.c46aa9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://608b11d906c99e56e9ff3678--gamut-preview.netlify.app>https://608b11d906c99e56e9ff3678--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/608b11d906c99e56e9ff3678>Deploy Logs</a></p>

**Pull request merged**